crates/factor-outbound-networking/src/lib.rs

```diff
@@ -92,8 +92,18 @@ impl Factor for OutboundNetworkingFactor {
             .expression_resolver()
             .clone();
         let allowed_hosts_future = async move {
-            let prepared = resolver.prepare().await?;
-            AllowedHostsConfig::parse(&hosts, &prepared)
+            let prepared = resolver.prepare().await.inspect_err(|err| {
+                tracing::error!(
```

Comment:
Does this result in a doubled error message at the console - once for the `tracing::error!` and once when the `?` bubbles back to the top and causes an error exit?

Reply:
It should do whatever it did before apart from the wording change on the `parse` path.